index.d.ts

```diff
@@ -0,0 +1,38 @@
+type TimeWindow = number; // e.g. 60 * 1000
+type Snapshot = object;
+
+export interface RollingWindowOptions {
+    numChunks: number;
```

Comment:
Each of options should be optional. For example `{ numChunks: 10 }` should be allowed as valid `options`.